examples/user-workload/1q

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
I'm confused about this file's name. Is it needed?

Reply:
I believe that was committed wrongly, possibly outcome of some generation or just my mistake, will remove.